ModelCatalogueCorePlugin/grails-app/assets/javascripts/modelcatalogue/core/orderedMapEnhancer.coffee

```diff
@@ -15,8 +15,7 @@ angular.module('mc.core.orderedMapEnhancer', ['mc.util.enhance']).config ['enhan
             if value.key == key
               value.value = newValue
               return newValue
-          # only push if the value is defined
-          self.values.push(key: key, value: newValue) if newValue or not key
+          self.values.push(key: key, value: newValue)
```

Comment:
yeah, exactly :-)

On Thu, Mar 24, 2016 at 12:52 PM Matouš Kučera <notifications@github.com>
wrote:

> In
> ModelCatalogueCorePlugin/grails-app/assets/javascripts/modelcatalogue/core/orderedMapEnhancer.coffee
> <https://github.com/MetadataRegistry/ModelCataloguePlugin/pull/535#discussion_r57306363>
> :
>
> > @@ -15,8 +15,7 @@ angular.module('mc.core.orderedMapEnhancer', ['mc.util.enhance']).config ['enhan
> >              if value.key == key
> >                value.value = newValue
> >                return newValue
> > -          # only push if the value is defined
> > -          self.values.push(key: key, value: newValue) if newValue or not key
> > +          self.values.push(key: key, value: newValue)
>
> Aha, now I see when the test is failing. Same directive is used for
> metadata and if I delete this if then it adds meta data like min max,
> etc...Am going to rollback this.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/MetadataRegistry/ModelCataloguePlugin/pull/535/files/2d8267218ddb1998ebd87b2d977e49cfc2116f7a#r57306363>
>